apps/extension/src/provider/Anoma.ts

```diff
@@ -0,0 +1,53 @@
+import { ChainInfo as Chain } from "@keplr-wallet/types";
+import { Anoma as IAnoma, Signer } from "@anoma/types";
+import { Ports, MessageRequester } from "../router";
+import {
+  GetChainMsg,
+  GetChainsMsg,
+  GetSignerMsg,
+  SuggestChainMsg,
+} from "../background/chains";
+
+export class Anoma implements IAnoma {
```

Comment:
I got the feeling this this might be better if it lived next to all the code the relates to the "core" part of the extension, `/src/background`. It could still be "provided" from here thought, if we want to have consumers get it through a provider.